app/controllers/admin/collections_controller.rb

```diff
@@ -252,24 +252,21 @@ def attach_poster
         @collection.poster.content = resized_poster
         @collection.poster.mime_type = 'image/png'
         @collection.poster.original_name = poster_file.original_filename
-        flash[:success] = "Poster file succesfully added."
+        flash[:success] = "Poster file successfully added."
       else
         flash[:error] = "Uploaded file is not a recognized poster image file"
       end
     else
       @collection.poster.content = ''
       @collection.poster.original_name = ''
-      flash[:success] = "Poster file succesfully removed."
+      flash[:success] = "Poster file successfully removed."
+      redirect_to admin_collection_path(@collection)
```

Comment:
I'm a little concerned about the chances of a double `redirect_to` when a delete fails to save.  Maybe it would be better to refactor the delete out to a separate action that is the same route but which is a DELETE instead of a POST?

Reply:
Yes, sounds good :+1: